src/confluence/subcommands/list.py

```diff
@@ -0,0 +1,114 @@
+import pandas as pd
```

Comment:
Please put together a docstring at the top of this file that shows some examples of how this subcommand is to be called/used. That would help provide context to review the rest of the file.